spec/xrootd-cmstfc.spec

```diff
@@ -29,12 +47,13 @@ Group: System Environment/Development
 %setup -q -c -n %{name}-%{version}
 
 %build
-
-%cmake -DCMAKE_BUILD_TYPE=RelWithDebInfo .
+cd %{name}-%{version}
+%cmake -DCMAKE_BUILD_TYPE=RelWithDebInfo -DCMAKE_INSTALL_LIBDIR=%{_lib} .
```

Comment:
Why are these lines necessary?

Reply:
Not sure, this changes were in the osg svn repo for god knows how long and there is no mention of them in the changelog

Reply:
I managed to get rid of the `cd %{name}-%{version}` by modifying the `%setup`.
The `DCMAKE_INSTALL_LIBDIR=%{_lib}` makes the files get installed into /usr/lib64, otherwise they go into /usr/lib

Reply:
I managed to get rid of the `cd %{name}-%{version}` by changing the `%setup%. 
`-DCMAKE_INSTALL_LIBDIR=%{_lib}` is needed so that files get installed into /usr/lib64 instead of /usr/lib.